scripts/machine-launch-script.sh

```diff
@@ -1,83 +1,260 @@
 #!/bin/bash
 
+#if CONDA_PREFIX already defined (because you already have a conda environment activated), just do the env creation in that prefix
+#the cmdline --prefix overrides this
+CONDA_INSTALL_PREFIX=/opt/conda
+CONDA_INSTALLER=https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh
```

Comment:
I feel like we should probably version this.

Reply:
```suggestion
CONDA_INSTALLER=https://github.com/conda-forge/miniforge/releases/download/4.11.0-4/Miniforge3-4.11.0-4-Linux-x86_64.sh
```
Sure, we can do this.